### ISSUE TITLE
AttributeError: 'str' object has no attribute 'decode'

### ISSUE THREAD

**Issue by BoudSTER** [ylab-hi/ScanITD#2]
Hello, 
I followed the installation instructions carefully and when I run the algorithm this error appears:

"""
(scanitd) MacBook-Pro-de-Boudry:ScanITD-master boudster$ python ScanITD.py -i Test/test.bam -r hg38.fa -o test
ScanITD build starts running: 2021-01-05 10:32:21
Checking for 'samtools': found /usr/local/bin/samtools

ScanITD build running done: 2021-01-05 10:32:21
ScanITD build takes 0.16238689422607422 seconds.
ScanITD call starts running: 2021-01-05 10:32:21
Traceback (most recent call last):
  File "ScanITD.py", line 552, in <module>
    main()
  File "ScanITD.py", line 537, in main
    itd_scan(input_bam=itd_build, output_prefix=options.output, ao_cutoff=options.ao, dp_cutoff=options.dp, vaf_cutoff=options.vaf, itd_len_cutoff=options.itd_len, target=options.target, mapq_cutoff=options.mapq, mismatch_cutoff=options.mismatch, fasta_file=options.ref)
  File "ScanITD.py", line 426, in itd_scan
    sr_id, sr_dict[sr_id], itd_id, itd_seq_dict[itd_id], mismatch_cutoff, fasta_file, input_bam)
  File "ScanITD.py", line 256, in mismatch_count
    dp_line = pysam.depth('{0}'.format(bamfile), '-q', '15', '-r', '{0}:{1}-{1}'.format(sr_pos[1], cov_pos+1)).decode('utf-8')
AttributeError: 'str' object has no attribute 'decode'
"""

Thank you for your time.

**Comment by dolittle007** [comment]
Thank you so much for letting me know about this bug.
I found that the latest versions of Pysam have changed their API, so it raised this error.
I fixed this bug by removing "decode('utf-8')".
I tested it, it worked.

**Comment by BoudSTER** [comment]
You are right, I had also made the correction .But I didn't know that the problem came from a version of Pysam ;).
Nevertheless by making this correction I get an empty vcf file with the test.bam file. 

I have decreased the vaf detect and we don't find the same vaf as the one indicated in the git. I find AB=0.063 whereas the vaf indicated in the git is AB=0.12

Is this normal ? or is this an error on my part ?
